client/my-sites/site-settings/site-tools/link.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+
+const SiteToolsLink = ( {
+	description,
+	href,
+	isWarning,
+	onClick,
+	title,
+} ) => {
+	const titleClasses = classNames( 'site-tools__section-title', {
+		'is-warning': isWarning,
+	} );
+
+	return (
+		<CompactCard
+			href={ href }
+			onClick={ onClick }
+			className="site-tools__link"
+		>
+			<div className="site-tools__content">
+				<p className={ titleClasses }>{ title }</p>
+				<p className="site-tools__section-desc">{ description }</p>
+			</div>
+		</CompactCard>
+	);
+};
+
+SiteToolsLink.propTypes = {
```

Comment:
Hmm, yes, I suppose you might what one without a title, and one without a description, and both would be fine. Maybe best to leave as is :)